cmd/migrate/migrate.go

```diff
@@ -80,6 +80,15 @@ func main() {
 		}
 	}
 
+	//removes unfavorited pages from all users in favorite pages tables
+	if tx.Migrator().HasTable(&models.FavoritePage{}) {
+		if err := tx.Where("favorite = ?", false).Delete(&models.FavoritePage{}); err != nil {
```

Comment:
I think this might just soft delete the entries: https://gorm.io/docs/delete.html#Delete-permanently